appveyor.yml

```diff
@@ -136,11 +99,7 @@ install:
 
   # Install the PyInstaller test dependencies.
   - '%CMD_IN_ENV% pip install -U --timeout 5 --retries 2
-        -r tests/requirements-tools.txt'
-  - 'if "%TEST_LIBRARIES%" == "YES" (
-        %CMD_IN_ENV% pip install -U --timeout 5 --retries 2
-             -r tests/requirements-libraries.txt
-    )'
+        -r tests/requirements-libraries.txt -r tests/requirements-tools.txt'
```

Comment:
Un-splitting the tests is a different change then "enable parallel testing" and should go into a separate commit. I suggest adding to the commit message something like "Now the test are running in parallel, the test-suite runs short/quick enough to not time-out. So we can unify the split test-runs." (If this is correct, of course, - which would be nice :-)